플레이어 클래스와 이동 관련 함수

sch9401 opened the pull request:
1차적인거라 대대적 개편 필요
다른 분 작성한거랑 합친 뒤 수정할 예정

sch9401 committed:
플레이어 클래스와 이동 관련 함수

1차적인거라 대대적 개편 필요

lshbluesky commented on Plyer_move.cpp:
```diff
@@ -0,0 +1,168 @@
+	//��ǻ� �÷��̾� ��ġ�� ǥ���ϸ� ��
+	void print() {					//ȭ�� ����
+
+		gotoxy(x, y);				//�÷��̾��� ��ġ�� �ڷ� ǥ��
+		cout << "��";
```
상단에 `#include "Text.hpp"` 를 추가하고 다음과 같이 수정하면
```suggestion
		gotoxy(x, y, "★");
```
`Text.hpp` 에 정의되어 있는 `gotoxy()` 함수를 사용할 수 있습니다.

lshbluesky commented on Plyer_move.cpp:
```diff
@@ -0,0 +1,168 @@
+int start_x, start_y; //���� ��ġ ���� �����
+
+//�Ʒ� �Լ� ������ �׳� main�� �����ϴ� ���� ���ҵ�, �ּ��� Randomstart�� Player�� �ؾ���
+int Playing() {
```
`Playing()` 함수에서 `Return` 구문이 없어서 함수의 반환형은 `void` 로 수정하는 것이 좋을 것 같아요.
```suggestion
void Playing() {
```

lshbluesky commented on Plyer_move.h:
```diff
@@ -0,0 +1,13 @@
+
+using namespace std;
+
+void cursor(int n);				//Ŀ�� ������ ����
+void gotoxy(int x, int y);		//�ܼ� �� ��ǥ�̵�
+int RandomStart(int min, int max);	//������ ��ġ ����
+bool isWall(int x, int y);			//���� �ִ��� Ȯ��
```
`Playing()` 함수의 원형 정의가 누락된 것 같아서 추가하면 좋을 것 같아요.
```suggestion
void Playing();
void cursor(int n);				//Ŀ�� ������ ����
int RandomStart(int min, int max);	//������ ��ġ ����
bool isWall(int x, int y);			//���� �ִ��� Ȯ��
```
또한, `gotoxy()` 함수는 `Text.hpp` 에 정의된 것으로 사용한다면
`void gotoxy(int x, int y);` 는 필요하지 않습니다.

lshbluesky committed:
[최적화] 소스 코드의 일부 함수 수정 및 최적화

- Text.hpp 에 정의되어 있는 gotoxy() 함수 사용.
- Playing() 함수의 반환형을 void 로 수정.
- 불필요한 gotoxy() 함수 원형 정의 제거.
- 헤더 파일의 확장자를 .hpp 로 변경.
- 소스 코드 파일을 유니코드(서명 있는 UTF-8) 로 인코딩 저장하여 한글 주석이 깨지는 문제 해결.

lshbluesky commented:
제가 아래의 항목을 수정해 보았습니다.
- [x] `Text.hpp` 에 정의되어 있는 `gotoxy()` 함수 사용.
- [X] `Playing()` 함수의 반환형을 `void` 로 수정.
- [X] 불필요한 `gotoxy()` 함수 원형 정의 제거.
- [X] 헤더 파일의 확장자를 `.hpp` 로 변경.
- [X] 소스 코드 파일을 `유니코드(서명 있는 UTF-8)` 로 인코딩 저장하여 한글 주석이 깨지는 문제 해결.
